cmd/opm/alpha/veneer/basic.go

```diff
@@ -21,10 +21,21 @@ func newBasicVeneerRenderCmd() *cobra.Command {
 	)
 	cmd := &cobra.Command{
 		Use:   "basic basic-veneer-file",
-		Short: "Generate a declarative config blob from a single 'basic veneer' file",
-		Long:  `Generate a declarative config blob from a single 'basic veneer' file, typified as a declarative configuration file where olm.bundle objects have no properties`,
-		Args:  cobra.ExactArgs(1),
+		Short: `Generate a declarative config blob from a single 'basic veneer' file
+When FILE is '-' or not provided, the veneer is read from standard input`,
+		Long:  `Generate a declarative config blob from a single 'basic veneer' file
+When FILE is '-' or not provided, the veneer is read from standard input`,
```

Comment:
It would be nice if -- since we're touching this file -- we could be consistent across both supported veneers.  There were many objections to "blob" for semver, and what we settled on was something like this:
```suggestion
		Short: `Generate a file-based catalog from a single 'basic veneer' file
When FILE is '-' or not provided, the veneer is read from standard input`,
		Long:  `Generate a file-based catalog from a single 'basic veneer' file
When FILE is '-' or not provided, the veneer is read from standard input`,
```